packages/integration-tests/src/integration/ui-behaviour/cluster-creation.spec.ts

```diff
@@ -31,9 +31,9 @@ describe('Assisted Installer UI behaviour - cluster creation', () => {
       clusterDetailsPage.openOpenshiftVersionDropdown();
 
       const expectedVersionIds = versionsFixtures.getExpectedVersionIds();
-      clusterDetailsPage.getOpenshiftVersionDropdown().find('li').each((versionItem, index) => {
-        expect(versionItem).to.have.id(expectedVersionIds[index]);
-      });
+      expectedVersionIds.forEach((id) => {
```

Comment:
I think the previous method was better because it guaranteed that the elements where shown in the expected order.

I suggest to change the original code to
```
      clusterDetailsPage.getOpenshiftVersionDropdown().find('[role="menuitem"]').each((versionItem, index) => {
        expect(versionItem).to.have.id(expectedVersionIds[index]);
      });
```

Reply:
That's a much less complicated approach. Thank you!